### PR TITLE
Package imguiml.v1.90.6

### PR DESCRIPTION
**Pull request by emilienlemaire**
### `imguiml.v1.90.6`
ImGUI bindings to OCaml
ImGUI bindings to OCaml

---
* Homepage: https://github.com/emilienlemaire/imguiml
* Source repo: git+https://github.com/emilienlemaire/imguiml.git
* Bug tracker: https://github.com/emilienlemaire/imguiml/issues

---
:camel: Pull-request generated by opam-publish v2.3.0

### CODE REVIEW

**Comment by shonfeder** [comment]
Hello! Sorry for the delay in reviewing.

Here's an overview of the current CI errors:

## arm64:

An error with the C bindings. Could you have a look? If it's out of scope for this package, you can mark it as unavailable on this architecture. See, e.g., https://github.com/ocaml/opam-repository/blob/7c9334f2a19f26566f35b9f442f843a69d5c0781/packages/ocaml-options-only-no-flat-float-array/ocaml-options-only-no-flat-float-array.1+bytecode-only/opam#L26

<details>

```
#=== ERROR while compiling imguiml.v1.90.6 ====================================#
# context              2.2.0~beta3~dev | linux/arm64 | ocaml-base-compiler.4.14.2 | pinned(https://github.com/emilienlemaire/imguiml/archive/refs/tags/v1.90.6.tar.gz)
# path                 ~/.opam/4.14/.opam-switch/build/imguiml.v1.90.6
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p imguiml -j 79 @install
# exit-code            1
# env-file             ~/.opam/log/imguiml-7-5eb8a0.env
# output-file          ~/.opam/log/imguiml-7-5eb8a0.out
### output ###
# (cd _build/default/cimgui/cimgui/build_static && /usr/bin/cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DIMGUI_STATIC=yes ..)
# -- The C compiler identification is GNU 12.2.0
# -- The CXX compiler identification is GNU 12.2.0
# -- Detecting C compiler ABI info
# -- Detecting C compiler ABI info - done
# -- Check for working C compiler: /usr/bin/cc - skipped
# -- Detecting C compile features
# -- Detecting C compile features - done
# -- Detecting CXX compiler ABI info
# -- Detecting CXX compiler ABI info - done
# -- Check for working CXX compiler: /usr/bin/c++ - skipped
# -- Detecting CXX compile features
# -- Detecting CXX compile features - done
# -- Configuring done
# -- Generating done
# -- Build files have been written to: /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/_build/default/cimgui/cimgui/build_static
# (cd _build/default/cimgui/cimgui/build_static && /usr/bin/make)
# [ 14%] Building CXX object CMakeFiles/cimgui.dir/cimgui.cpp.o
# [ 28%] Building CXX object CMakeFiles/cimgui.dir/imgui/imgui.cpp.o
# [ 42%] Building CXX object CMakeFiles/cimgui.dir/imgui/imgui_demo.cpp.o
# [ 57%] Building CXX object CMakeFiles/cimgui.dir/imgui/imgui_draw.cpp.o
# [ 71%] Building CXX object CMakeFiles/cimgui.dir/imgui/imgui_tables.cpp.o
# [ 85%] Building CXX object CMakeFiles/cimgui.dir/imgui/imgui_widgets.cpp.o
# [100%] Linking CXX static library cimgui.a
# [100%] Built target cimgui
# (cd _build/default/cimgui/cimgui/build_dynamic && /usr/bin/cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo ..)
# -- The C compiler identification is GNU 12.2.0
# -- The CXX compiler identification is GNU 12.2.0
# -- Detecting C compiler ABI info
# -- Detecting C compiler ABI info - done
# -- Check for working C compiler: /usr/bin/cc - skipped
# -- Detecting C compile features
# -- Detecting C compile features - done
# -- Detecting CXX compiler ABI info
# -- Detecting CXX compiler ABI info - done
# -- Check for working CXX compiler: /usr/bin/c++ - skipped
# -- Detecting CXX compile features
# -- Detecting CXX compile features - done
# -- Configuring done
# -- Generating done
# -- Build files have been written to: /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/_build/default/cimgui/cimgui/build_dynamic
# (cd _build/default/cimgui/cimgui/build_dynamic && /usr/bin/make)
# [ 14%] Building CXX object CMakeFiles/cimgui.dir/cimgui.cpp.o
# [ 28%] Building CXX object CMakeFiles/cimgui.dir/imgui/imgui.cpp.o
# [ 42%] Building CXX object CMakeFiles/cimgui.dir/imgui/imgui_demo.cpp.o
# [ 57%] Building CXX object CMakeFiles/cimgui.dir/imgui/imgui_draw.cpp.o
# [ 71%] Building CXX object CMakeFiles/cimgui.dir/imgui/imgui_tables.cpp.o
# [ 85%] Building CXX object CMakeFiles/cimgui.dir/imgui/imgui_widgets.cpp.o
# [100%] Linking CXX shared library cimgui.so
# [100%] Built target cimgui
# (cd _build/default/src/impl/backend/build_static && /usr/bin/cmake .. -DIMGUI_STATIC=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo)
# -- The C compiler identification is GNU 12.2.0
# -- The CXX compiler identification is GNU 12.2.0
# -- Detecting C compiler ABI info
# -- Detecting C compiler ABI info - done
# -- Check for working C compiler: /usr/bin/cc - skipped
# -- Detecting C compile features
# -- Detecting C compile features - done
# -- Detecting CXX compiler ABI info
# -- Detecting CXX compiler ABI info - done
# -- Check for working CXX compiler: /usr/bin/c++ - skipped
# -- Detecting CXX compile features
# -- Detecting CXX compile features - done
# -- Found GLEW: /usr/include (found version "2.2.0") 
# -- Configuring done
# -- Generating done
# -- Build files have been written to: /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/_build/default/src/impl/backend/build_static
# (cd _build/default/src/impl/backend/build_static && /usr/bin/make)
# [ 33%] Building CXX object CMakeFiles/cimgui_impl.dir/imgui_impl_opengl3.cpp.o
# [ 66%] Building CXX object CMakeFiles/cimgui_impl.dir/imgui_impl_glfw.cpp.o
# [100%] Linking CXX static library libcimgui_impl.a
# [100%] Built target cimgui_impl
# (cd _build/default/src/impl/backend/build_dynamic && /usr/bin/cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo)
# -- The C compiler identification is GNU 12.2.0
# -- The CXX compiler identification is GNU 12.2.0
# -- Detecting C compiler ABI info
# -- Detecting C compiler ABI info - done
# -- Check for working C compiler: /usr/bin/cc - skipped
# -- Detecting C compile features
# -- Detecting C compile features - done
# -- Detecting CXX compiler ABI info
# -- Detecting CXX compiler ABI info - done
# -- Check for working CXX compiler: /usr/bin/c++ - skipped
# -- Detecting CXX compile features
# -- Detecting CXX compile features - done
# -- Found GLEW: /usr/include (found version "2.2.0") 
# -- Configuring done
# -- Generating done
# -- Build files have been written to: /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/_build/default/src/impl/backend/build_dynamic
# (cd _build/default/src/impl/backend/build_dynamic && /usr/bin/make)
# [ 33%] Building CXX object CMakeFiles/cimgui_impl.dir/imgui_impl_opengl3.cpp.o
# [ 66%] Building CXX object CMakeFiles/cimgui_impl.dir/imgui_impl_glfw.cpp.o
# [100%] Linking CXX shared library libcimgui_impl.so
# [100%] Built target cimgui_impl
# File "src/dune", line 335, characters 9-34:
# 335 |   (names cimgui_bindings_generated)
#                ^^^^^^^^^^^^^^^^^^^^^^^^^
# (cd _build/default/src && /usr/bin/gcc -fPIC -I/home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui -DCIMGUI_DEFINE_ENUMS_AND_STRUCTS=1 -Wno-incompatible-pointer-types -Wno-discarded-qualifiers -g -I /home/opam/.opam/4.14/lib/ocaml -I /home/opam/.opam/4.14/lib/bigarray-compat -I /home/opam/.opam/4.14/lib/ctypes -I /home/opam/.opam/4.14/lib/ctypes/stubs -I /home/opam/.opam/4.14/lib/integers -I /home/opam/.opam/4.14/lib/stdlib-shims -o cimgui_bindings_generated.o -c cimgui_bindings_generated.c)
# cimgui_bindings_generated.c: In function 'cimgui_binding__136_igTextV':
# cimgui_bindings_generated.c:844:18: error: incompatible type for argument 2 of 'igTextV'
#   844 |    igTextV(x603, x604);
#       |                  ^~~~
#       |                  |
#       |                  void *
# In file included from cimgui_bindings_generated.c:5:
# /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui/cimgui.h:3511:49: note: expected 'va_list' but argument is of type 'void *'
#  3511 | CIMGUI_API void igTextV(const char* fmt,va_list args);
#       |                                         ~~~~~~~~^~~~
# cimgui_bindings_generated.c: In function 'cimgui_binding__138_igTextColoredV':
# cimgui_bindings_generated.c:861:31: error: incompatible type for argument 3 of 'igTextColoredV'
#   861 |    igTextColoredV(x618, x619, x620);
#       |                               ^~~~
#       |                               |
#       |                               void *
# /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui/cimgui.h:3513:73: note: expected 'va_list' but argument is of type 'void *'
#  3513 | CIMGUI_API void igTextColoredV(const ImVec4 col,const char* fmt,va_list args);
#       |                                                                 ~~~~~~~~^~~~
# cimgui_bindings_generated.c: In function 'cimgui_binding__140_igTextDisabledV':
# cimgui_bindings_generated.c:874:26: error: incompatible type for argument 2 of 'igTextDisabledV'
#   874 |    igTextDisabledV(x627, x628);
#       |                          ^~~~
#       |                          |
#       |                          void *
# /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui/cimgui.h:3515:57: note: expected 'va_list' but argument is of type 'void *'
#  3515 | CIMGUI_API void igTextDisabledV(const char* fmt,va_list args);
#       |                                                 ~~~~~~~~^~~~
# cimgui_bindings_generated.c: In function 'cimgui_binding__142_igTextWrappedV':
# cimgui_bindings_generated.c:887:25: error: incompatible type for argument 2 of 'igTextWrappedV'
#   887 |    igTextWrappedV(x635, x636);
#       |                         ^~~~
#       |                         |
#       |                         void *
# /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui/cimgui.h:3517:56: note: expected 'va_list' but argument is of type 'void *'
#  3517 | CIMGUI_API void igTextWrappedV(const char* fmt,va_list args);
#       |                                                ~~~~~~~~^~~~
# cimgui_bindings_generated.c: In function 'cimgui_binding__144_igLabelTextV':
# cimgui_bindings_generated.c:902:29: error: incompatible type for argument 3 of 'igLabelTextV'
#   902 |    igLabelTextV(x646, x647, x648);
#       |                             ^~~~
#       |                             |
#       |                             void *
# /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui/cimgui.h:3519:72: note: expected 'va_list' but argument is of type 'void *'
#  3519 | CIMGUI_API void igLabelTextV(const char* label,const char* fmt,va_list args);
#       |                                                                ~~~~~~~~^~~~
# cimgui_bindings_generated.c: In function 'cimgui_binding__146_igBulletTextV':
# cimgui_bindings_generated.c:915:24: error: incompatible type for argument 2 of 'igBulletTextV'
#   915 |    igBulletTextV(x655, x656);
#       |                        ^~~~
#       |                        |
#       |                        void *
# /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui/cimgui.h:3521:55: note: expected 'va_list' but argument is of type 'void *'
#  3521 | CIMGUI_API void igBulletTextV(const char* fmt,va_list args);
#       |                                               ~~~~~~~~^~~~
# cimgui_bindings_generated.c: In function 'cimgui_binding__215_igTreeNodeV_Str':
# cimgui_bindings_generated.c:2208:48: error: incompatible type for argument 3 of 'igTreeNodeV_Str'
#  2208 |    _Bool x1952 = igTreeNodeV_Str(x1949, x1950, x1951);
#       |                                                ^~~~~
#       |                                                |
#       |                                                void *
# /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui/cimgui.h:3590:76: note: expected 'va_list' but argument is of type 'void *'
#  3590 | CIMGUI_API bool igTreeNodeV_Str(const char* str_id,const char* fmt,va_list args);
#       |                                                                    ~~~~~~~~^~~~
# cimgui_bindings_generated.c: In function 'cimgui_binding__216_igTreeNodeV_Ptr':
# cimgui_bindings_generated.c:2217:48: error: incompatible type for argument 3 of 'igTreeNodeV_Ptr'
#  2217 |    _Bool x1959 = igTreeNodeV_Ptr(x1956, x1957, x1958);
#       |                                                ^~~~~
#       |                                                |
#       |                                                void *
# /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui/cimgui.h:3591:76: note: expected 'va_list' but argument is of type 'void *'
#  3591 | CIMGUI_API bool igTreeNodeV_Ptr(const void* ptr_id,const char* fmt,va_list args);
#       |                                                                    ~~~~~~~~^~~~
# cimgui_bindings_generated.c: In function 'cimgui_binding__220_igTreeNodeExV_Str':
# cimgui_bindings_generated.c:2252:57: error: incompatible type for argument 4 of 'igTreeNodeExV_Str'
#  2252 |    _Bool x1995 = igTreeNodeExV_Str(x1989, x1990, x1993, x1994);
#       |                                                         ^~~~~
#       |                                                         |
#       |                                                         void *
# /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui/cimgui.h:3595:103: note: expected 'va_list' but argument is of type 'void *'
#  3595 | CIMGUI_API bool igTreeNodeExV_Str(const char* str_id,ImGuiTreeNodeFlags flags,const char* fmt,va_list args);
#       |                                                                                               ~~~~~~~~^~~~
# cimgui_bindings_generated.c: In function 'cimgui_binding__221_igTreeNodeExV_Ptr':
# cimgui_bindings_generated.c:2262:57: error: incompatible type for argument 4 of 'igTreeNodeExV_Ptr'
#  2262 |    _Bool x2006 = igTreeNodeExV_Ptr(x2000, x2001, x2004, x2005);
#       |                                                         ^~~~~
#       |                                                         |
#       |                                                         void *
# /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui/cimgui.h:3596:103: note: expected 'va_list' but argument is of type 'void *'
#  3596 | CIMGUI_API bool igTreeNodeExV_Ptr(const void* ptr_id,ImGuiTreeNodeFlags flags,const char* fmt,va_list args);
#       |                                                                                               ~~~~~~~~^~~~
# cimgui_bindings_generated.c: In function 'cimgui_binding__254_igSetTooltipV':
# cimgui_bindings_generated.c:2637:25: error: incompatible type for argument 2 of 'igSetTooltipV'
#  2637 |    igSetTooltipV(x2353, x2354);
#       |                         ^~~~~
#       |                         |
#       |                         void *
# /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui/cimgui.h:3629:55: note: expected 'va_list' but argument is of type 'void *'
#  3629 | CIMGUI_API void igSetTooltipV(const char* fmt,va_list args);
#       |                                               ~~~~~~~~^~~~
# cimgui_bindings_generated.c: In function 'cimgui_binding__257_igSetItemTooltipV':
# cimgui_bindings_generated.c:2655:29: error: incompatible type for argument 2 of 'igSetItemTooltipV'
#  2655 |    igSetItemTooltipV(x2363, x2364);
#       |                             ^~~~~
#       |                             |
#       |                             void *
# /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui/cimgui.h:3632:59: note: expected 'va_list' but argument is of type 'void *'
#  3632 | CIMGUI_API void igSetItemTooltipV(const char* fmt,va_list args);
#       |                                                   ~~~~~~~~^~~~
# cimgui_bindings_generated.c: In function 'cimgui_binding__312_igLogTextV':
# cimgui_bindings_generated.c:3013:22: error: incompatible type for argument 2 of 'igLogTextV'
#  3013 |    igLogTextV(x2683, x2684);
#       |                      ^~~~~
#       |                      |
#       |                      void *
# /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui/cimgui.h:3687:52: note: expected 'va_list' but argument is of type 'void *'
#  3687 | CIMGUI_API void igLogTextV(const char* fmt,va_list args);
#       |                                            ~~~~~~~~^~~~
# cimgui_bindings_generated.c: In function 'cimgui_binding__469_ImGuiTextBuffer_appendfv':
# cimgui_bindings_generated.c:4104:43: error: incompatible type for argument 3 of 'ImGuiTextBuffer_appendfv'
#  4104 |    ImGuiTextBuffer_appendfv(x3476, x3477, x3478);
#       |                                           ^~~~~
#       |                                           |
#       |                                           void *
# /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui/cimgui.h:3843:88: note: expected 'va_list' but argument is of type 'void *'
#  3843 | CIMGUI_API void ImGuiTextBuffer_appendfv(ImGuiTextBuffer* self,const char* fmt,va_list args);
#       |                                                                                ~~~~~~~~^~~~
# cimgui_bindings_generated.c: In function 'cimgui_binding__685_igImFormatStringV':
# cimgui_bindings_generated.c:6603:55: error: incompatible type for argument 4 of 'igImFormatStringV'
#  6603 |    int x5766 = igImFormatStringV(x5760, x5761, x5764, x5765);
#       |                                                       ^~~~~
#       |                                                       |
#       |                                                       void *
# /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui/cimgui.h:4059:84: note: expected 'va_list' but argument is of type 'void *'
#  4059 | CIMGUI_API int igImFormatStringV(char* buf,size_t buf_size,const char* fmt,va_list args);
#       |                                                                            ~~~~~~~~^~~~
# cimgui_bindings_generated.c: In function 'cimgui_binding__687_igImFormatStringToTempBufferV':
# cimgui_bindings_generated.c:6625:55: error: incompatible type for argument 4 of 'igImFormatStringToTempBufferV'
#  6625 |    igImFormatStringToTempBufferV(x5778, x5779, x5780, x5781);
#       |                                                       ^~~~~
#       |                                                       |
#       |                                                       void *
# /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui/cimgui.h:4061:117: note: expected 'va_list' but argument is of type 'void *'
#  4061 | CIMGUI_API void igImFormatStringToTempBufferV(const char** out_buf,const char** out_buf_end,const char* fmt,va_list args);
#       |                                                                                                             ~~~~~~~~^~~~
# cimgui_bindings_generated.c: In function 'cimgui_binding__1340_igDebugLogV':
# cimgui_bindings_generated.c:12160:24: error: incompatible type for argument 2 of 'igDebugLogV'
# 12160 |    igDebugLogV(x10510, x10511);
#       |                        ^~~~~~
#       |                        |
#       |                        void *
# /home/opam/.opam/4.14/.opam-switch/build/imguiml.v1.90.6/cimgui/cimgui/cimgui.h:4714:53: note: expected 'va_list' but argument is of type 'void *'
#  4714 | CIMGUI_API void igDebugLogV(const char* fmt,va_list args);
#       |                                             ~~~~~~~~^~~~



<><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
+- The following actions failed
| - build imguiml v1.90.6
+- 
```

</details>

## debian-10 ocaml-4.14

Failing due to the older cmake version avaiable in the debian 10 repository. Do you reckon it would be bossible to support CMake 3.14.4 for your build?

<details>

```
[ERROR] The compilation of imguiml.v1.90.6 failed at "dune build -p imguiml -j 39 @install".

#=== ERROR while compiling imguiml.v1.90.6 ====================================#
# context              2.2.0~beta3~dev | linux/x86_64 | ocaml-base-compiler.4.14.2 | pinned(https://github.com/emilienlemaire/imguiml/archive/refs/tags/v1.90.6.tar.gz)
# path                 ~/.opam/4.14/.opam-switch/build/imguiml.v1.90.6
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p imguiml -j 39 @install
# exit-code            1
# env-file             ~/.opam/log/imguiml-7-c358f7.env
# output-file          ~/.opam/log/imguiml-7-c358f7.out
### output ###
# File "cimgui/dune", line 3, characters 0-630:
#  3 | (rule
#  4 |  (deps
#  5 |   (source_tree cimgui))
# ....
# 26 |       (run make)))
# 27 |     (copy cimgui/build_dynamic/cimgui.so dllcimgui.so)
# 28 |     (copy cimgui/build_static/cimgui.a libcimgui.a)))))
# (cd _build/default/cimgui/cimgui/build_static && /usr/bin/cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DIMGUI_STATIC=yes ..)
# -- Configuring incomplete, errors occurred!
# CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
#   CMake 3.15 or higher is required.  You are running version 3.13.4
# 
# 



<><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
+- The following actions failed
| - build imguiml v1.90.6
```

</details>

### FreeBSD

This appears to be an instance of https://github.com/ocaml/opam/issues/5923, and as such unrelated to this package.

<details>
```
#=== ERROR while compiling conf-glew.1 ========================================#
# context              2.2.0~beta3~dev | freebsd/x86_64 | ocaml-base-compiler.4.14.2 | file:///home/opam/opam-repository
# path                 ~/.opam/4.14.2/.opam-switch/build/conf-glew.1
# command              /usr/local/bin/pkg-config --print-errors --exists glew
# exit-code            1
# env-file             ~/.opam/log/conf-glew-37637-570a7c.env
# output-file          ~/.opam/log/conf-glew-37637-570a7c.out
### output ###
# Package glew was not found in the pkg-config search path.
# Perhaps you should add the directory containing `glew.pc'
# to the PKG_CONFIG_PATH environment variable
# Package 'glew' not found



<><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
┌─ The following actions failed
│ λ build conf-glew 1
```

</details>

**Comment by emilienlemaire** [comment]
Hi,  thank you for taking the time to make a review :smile:

The last commit should handle most of the problems.

For FreeBSD, the problem should be handled on the `conf-glew` package side, as it is the one not building, and `glew` is listed as one of the depext of the package

File: packages/imguiml/imguiml.v1.90.6/opam [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  "ctypes-foreign" {>= "0.21.1"}
+  "odoc" {with-doc}
+  "sherlodoc" {with-doc}
+  "ppx_expect" {with-test}
+  "conf-glew"
+  "conf-glfw3"
+]
+depexts: [
+  ["gcc" "cmake"]
+]
```
Comment (mseri) [inline packages/imguiml/imguiml.v1.90.6/opam:31]:
```suggestion
]
```
A c compiler is already installed, the depexts as they are are not really used.
If you want to enforce `gcc` use `"conf-gcc"` as a dependency and if you want to enforce the presence of `clang` use `"conf-clang"` as a dependency

File: packages/imguiml/imguiml.v1.90.6/opam [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "3.15"}
+  "goblint-cil" {>= "2.0.0"}
+  "ctypes"
+  "ctypes-foreign" {>= "0.21.1"}
+  "odoc" {with-doc}
+  "sherlodoc" {with-doc}
+  "ppx_expect" {with-test}
+  "conf-glew"
```
Comment (mseri) [inline packages/imguiml/imguiml.v1.90.6/opam:26]:
```suggestion
  "conf-cmake"
  "conf-glew"
```

**Comment by mseri** [comment]
Thanks

**Pull request merged**